ruby/extractor/src/extractor.rs

```diff
@@ -26,6 +26,7 @@ pub struct Options {
 
 pub fn run(options: Options) -> std::io::Result<()> {
     extractor::set_tracing_level("ruby");
+    tracing::info!("Starting extraction");
```

Comment:
Perhaps "Extraction started" to match the style of the log message at the end.

Reply:
Thanks; I have updated the message.